.github/workflows/e2e-arm64-template.yaml

```diff
@@ -1,8 +1,14 @@
 ---
-name: E2E-arm64
+name: Reusable Arm64 E2E Workflow
```

Comment:
Probably I am missing something, the template is only used by e2e-arm64-nightly for now. In other words, the template has only one user, then why do you need to create a template? If the template can be reused by both ARM e2e test and NON-ARM e2e test, then it makes sense.

Reply:
We use a re-usable workflow so we can run the same template for `main` and `release-3.5` with different parameters. Similar to how robustness workflows run across different branches using the same workflow template.

Reply:
thx for the clarification. It seems it would be better if there is a document for all the workflow tests.